src/displayapp/fonts/CMakeLists.txt

```diff
@@ -1,19 +1,34 @@
-cmake_minimum_required(VERSION 3.10)
-
 set(FONTS jetbrains_mono_42 jetbrains_mono_76 jetbrains_mono_bold_20
    jetbrains_mono_extrabold_compressed lv_font_navi_80 lv_font_sys_48
    open_sans_light)
-find_program(LV_FONT_CONV "lv_font_conv" NO_CACHE REQUIRED)
+find_program(LV_FONT_CONV "lv_font_conv" NO_CACHE REQUIRED
+   HINTS "${CMAKE_SOURCE_DIR}/node_modules/.bin")
```

Comment:
I like this better then trusting PATH. once it's here, may as well add the global node_modules bin's too.

Reply:
do you know where this global `node_modules` folder is?

Reply:
seems to be `$HOME/.local/lib64/node_modules`, and the binary ended up in `$HOME/.local/bin/lv_font_conv` too.

Reply:
is that path in the global `PATH` variable? Maybe already added by `npm`? Then we won't need to add a `HINT` for it

Reply:
`.local/bin` should be, but sometimes isn't. however, it is generic enough that trusting that it is seems fine to me.